CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Table of Contents
 
+- [v0.63.0](#v0630)
+- [v0.62.0](#v0620)
```

Comment:
We also need to define these links. This is currently being done at the very bottom of this file.